Makefile

```diff
@@ -9,16 +11,16 @@ docker:
 	docker cp ytl-linux-builder:/app/$(IMAGE) .
 
 create-vb-vm:
-	-VBoxManage controlvm $(VB_VMNAME) poweroff
-	-VBoxManage unregistervm $(VB_VMNAME) --delete
-	VBoxManage createvm --name $(VB_VMNAME) --register --ostype Linux_64
-	VBoxManage modifyvm $(VB_VMNAME) --memory 2048 --nic1 nat --usb off --firmware efi --cpus 2 --vram 16
+	-VBoxManage controlvm "$(VM_NAME)" poweroff
+	-VBoxManage unregistervm "$(VM_NAME)" --delete
+	VBoxManage createvm --name "$(VM_NAME)" --register --ostype Linux_64
+	VBoxManage modifyvm "$(VM_NAME)" --memory $(VM_DISK_SIZE) --nic1 nat --usb off --firmware efi --cpus 2 --vram 16
```

Comment:
Should VM_DISK_SIZE be VM_MEMORY_SIZE?

Reply:
Absolutely! It appears VirtualBox limits the memory size according to the current host. In my case I run

`VBoxManage modifyvm "YTL Linux" --memory 51200 --nic1 nat --usb off --firmware efi --cpus 2 --vram 16`

but the virtual machine gets "only" 32 Gb RAM.

Do you want to make a pull request or shall I fix this?

Reply:
You can fix it.